beetmoverscript/constants.py

```diff
@@ -8,8 +9,8 @@
     '.dmg': 'application/x-iso9660-image',
     '.json': 'application/json',
     '.mar': 'application/octet-stream',
+    '.msi': 'application/x-msi',
```

Comment:
@Callek is this the same with `application/x-msdownload`?

Reply:
Googling made me only find `/x-msi` as appropriate since /x-msdownload is more for dll and exe http://mimeapplication.net/x-msdownload

I also have /x-msi on my Ubuntu 18.04 `/etc/mime.types`

Reply:
I have `application/x-msdownload` on my macbook but I guess we can keep yours instead. It doesn't matter what we have as long as it's something valid that makes the presigned url function generation happy.  If we have to, we can always address later.